app/controllers/comments_controller.rb

```diff
@@ -3,7 +3,6 @@ class CommentsController < ApplicationController
 
   # GET /comments or /comments.json
   def index
-    @comments = Comment.all
```

Comment:
This is superflouous and actually also wrong, because `load_and_authorize_resource` already does:

```ruby
@comments = Comment.accessible_by(current_ability)
```

which filters down the list to only the ones the current user is able to see.

Reply:
great!